config/application.yml.default

```diff
@@ -445,7 +445,7 @@ production:
   participate_in_dap: true
   password_pepper:
   piv_cac_verify_token_secret:
-  platform_authentication_enabled: false
+  platform_auth_set_up_enabled: true
```

Comment:
If we're not overriding the value to be something different from the default, we could leave it out entirely.

```suggestion
```